0_RootFS/llvm_common.jl

```diff
@@ -87,7 +90,9 @@ function llvm_script(;version = v"8.0.1", llvm_build_type = "Release", kwargs...
     CMAKE_FLAGS+=(-DLLVM_ENABLE_CXX1Y=ON -DLLVM_ENABLE_PIC=ON)
 
     # tell libcxx to use compiler-rt
-    CMAKE_FLAGS+=(-DLIBCXX_USE_COMPILER_RT=ON)
+    if [[ "${LLVM_MAJ_VER}" == "9" ]]; then
+        CMAKE_FLAGS+=(-DLIBCXX_USE_COMPILER_RT=ON)
+    fi
```

Comment:
Out of curiosity, why do we want to use `compiler-rt` for `libcxx` only on `v9`?